tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,8 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel.
+
+require('source-map-support').install();
+require('babel/register')(require('./babel-config.js')); // #RemoveInProd this line is removed in isopack.js
```

Comment:
This suggestion doesn't necessarily need to block this PR, but I think it would be better if we wrote our own `require` hook, and used our own caching system. It should be pretty straightforward (see https://github.com/babel/babel/blob/master/src/babel/api/register/node.js), and I think we could do a lot better than `require("babel/register")`—for one thing, it doesn't even hash file contents when generating the `cacheKey`.